### ISSUE TITLE
Edit Question: ua_ruhr_v1.1.0-beta.1/general/funding-funder/name

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#50]
**Describe the request**  
Proposal for a more specific wording of the question

**Propose what exactly should be changed**  
Concretize "Who" for example in organization / sponsor (Organisation / Geldgeber)

**Original setting or content**   
*question text ger*  
"Wer fördert das Projekt?"

**Proposed setting or content**   
What should it look like in the new version, e.g. 
*question text ger*  
„Welche Organisationen / Geldgeber fördert das Projekt?“

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [ ] question text (en and ger)

**Comment by smohex** [comment]
TUDO: Not a good formulation; "who" is actually the most neutral; if reworded, then rather "which institutions or organizations...".
UDE: I agree to leave the wording as it is. Therefore ticket closed
